packages/shipjs/src/step/prepare/__tests__/createPullRequest.spec.js

```diff
@@ -69,15 +69,8 @@ describe('createPullRequest', () => {
       .mockImplementation(({ version, type }) => `# v${version} (${type})`);
     const mockFormatPullRequestMessage = jest
       .fn()
-      .mockImplementation(
-        ({ formatPullRequestTitle, nextVersion, releaseType }) => {
-          return [
-            formatPullRequestTitle({
```

Comment:
why this change by the way? was the formatting not needed in the test?

Reply:
I don't know if/how this was ever working!

In `createPullRequest.js` `formatPullRequestTitle` is never passed to `formatPullRequestMessage` and so `formatPullRequestTitle` is `undefined` and not able to be called. The relevant bit is:
```js
    const title = formatPullRequestTitle({ version: nextVersion, releaseType });
    const message = formatPullRequestMessage({
      repo,
      repoURL,
      baseBranch,
      stagingBranch,
      destinationBranch: baseBranch,
      releaseType,
      diffURL,
      currentVersion,
      nextVersion,
      publishCommands,
      title,
    });
```

It doesn't make any sense that it was ever working.